test/proximity.js

```diff
@@ -1049,6 +1049,86 @@ exports["Proximity: HCSR04"] = {
   }
 };
 
+exports["Proximity: HCSR04I2C"] = {
+  setUp: function(done) {
+    this.board = newBoard();
+    this.clock = sinon.useFakeTimers();
+    this.i2cConfig = sinon.spy(MockFirmata.prototype, "i2cConfig");
+    this.i2cReadOnce = sinon.stub(MockFirmata.prototype, "i2cReadOnce", function(ADDRESS, BYTES_TO_READ, callback) {
+      // Use this for 1000 us as duration (HIGH and LOW)
+      callback([3, 232]);
```

Comment:
The way this is handled in LidarLite makes it more clear what's happening. 
````
var val = 1000;
callback([ val >> 8, val & 0xff ])
````
Sorry for my bad suggestion earlier.